src/iuse_actor.cpp

```diff
@@ -235,6 +237,10 @@ cata::optional<int> iuse_transform::use( Character &p, item &it, bool t, const t
         p.add_msg_if_player( m_info, _( "You need to wield the %1$s before activating it." ), it.tname() );
         return cata::nullopt;
     }
+    if( possess && need_empty && !it.empty() ) {
```

Comment:
Is the possess check necessary here? It looks like "need_empty" will block ALLOWS_REMOTE_USE flag and you don't have this flag on the boxes if I read it correctly

Reply:
You're right, it looks like that check was unnecessary